### ISSUE TITLE
Undo user post answer from activity newsfeed fails with Cannot read property 'option_id' of undefined

### ISSUE THREAD

**Issue by petervojtek** [PowerlineApp/powerline-mobile#180]
When post is answered (upvoted/downvoted), Undo button is displayed in activity newsfeed item.
clicking the Undo button fails with 

```
ionic.bundle.min.js:139 TypeError: Cannot read property 'option_id' of undefined
    at n.$scope.unsign (home.js:241)
    at fn (eval at compile (ionic.bundle.min.js:243), <anonymous>:4:209)
    at ionic.bundle.min.js:446
    at n.$eval (ionic.bundle.min.js:165)
    at n.$apply (ionic.bundle.min.js:166)
    at HTMLAnchorElement.<anonymous> (ionic.bundle.min.js:446)
    at HTMLAnchorElement.dispatch (jquery.js:4435)
    at HTMLAnchorElement.r.handle (jquery.js:4121)
    at n (ionic.bundle.min.js:22)
    at t (ionic.bundle.min.js:22)
    at HTMLDocument.r (ionic.bundle.min.js:22)
```

unsigning from user post detail works properly.

**Comment by petervojtek** [comment]
blocked by #187

**Comment by petervojtek** [comment]
already fixed in #187
